src/Microsoft.TestPlatform.PlatformAbstractions/Interfaces/System/PlatformArchitecture.cs

```diff
@@ -13,4 +13,5 @@ public enum PlatformArchitecture
     ARM,
     ARM64,
     S390x,
+    PPC64le,
```

Comment:
Maybe name this `Ppc64le` to keep it consistent with dotnet/runtime?

Reply:
Agree. Will change